main.py

```diff
@@ -84,17 +85,27 @@ def get_selenium_driver():
     driver = webdriver.Firefox(options=options, service=driver_service)
     return driver
 
+
 def send_message(token, chat_id, msg):
     url = f"https://api.telegram.org/bot{token}/sendMessage?chat_id={chat_id}&text={msg}"
     requests.get(url).json()
 
-def process(db_path, args):
+
+def discarded_pub(pub, filters):
+    for word in filters:
+        if word in pub.description or word in pub.location:
```

Comment:
I would convert the text to lowercase before comparing so capitalization or uppercase text gets filtered.

Reply:
done